lms/djangoapps/instructor/static/instructor/ProblemBrowser/components/Main/Main.jsx

```diff
@@ -22,33 +24,57 @@ export default class Main extends React.Component {
     this.setState({ showDropdown: false });
   }
 
+  initiateReportGeneration() {
+    this.props.createProblemResponsesReportTask(
+      this.props.problemResponsesEndpoint,
+      this.props.taskStatusEndpoint,
+      this.props.selectedBlock,
+    );
+  }
+
   render() {
     const { selectedBlock, onSelectBlock } = this.props;
 
     return (
-      <div className="problem-browser">
-        <Button onClick={this.handleToggleDropdown} label={gettext('Select a section or problem')} />
-        <input type="text" name="problem-location" value={selectedBlock} disabled />
-        {this.state.showDropdown &&
-        <BlockBrowser onSelectBlock={(blockId) => {
-          this.hideDropdown();
-          onSelectBlock(blockId);
-        }}
-        />}
+      <div className="problem-browser-container">
+        <div className="problem-browser">
+          <Button
+            onClick={this.handleToggleDropdown}
+            label={gettext('Select a section or problem')}
+          />
+          <input type="text" name="problem-location" value={selectedBlock} disabled />
```

Comment:
Just noticed: I think this input needs an `aria-labelledby` or something to better hint to users with screen readers what it's for.

Reply:
This input is disabled, so doesn't show up in ARIA. I don't think there is too much of value here since this can't be edited, and is mainly there to show the id. The only use here is to perhaps copy the block id. For that I think it would make more sense to add an explicit button.